src/ArcgisAccount/ArcgisAccount.js

```diff
@@ -120,7 +121,7 @@ class ArcgisAccount extends Component {
           />
         }
         open={this.state.open}
-        onRequestClose={this.closeAccountControl}
+        onRequestClose={onRequestClose || this.closeAccountControl}
```

Comment:
I know this is just a workaround, but it might be useful to pass the default close method to the custom `onRequestClose`. So something like:
```
onRequestClose={() => {
  onRequestClose ? onRequestClose(this.closeAccountControl) : this.closeAccountControl()
}
```
This would give you the option to still close the modal based on some custom logic.

Reply:
Good idea. I have updated the PR.